config/db/mysql-upgrade-user-table-3.4.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE user MODIFY tmpblocktimestamp BIGINT;
+ALTER TABLE user MODIFY tmpblockattempts integer;
```

Comment:
Converting tmpblocktimestamp does not give an error, but converts the value to BIGINT:
DATETIME `2022-01-02 03:04:05` is converted to BIGINT `20220102030405`. This means in practice that any tempblock that is not NULL will become active because it will be >> than the current time_t.

A tmpblockattempts that is wrongly configured as a datetime wouldn't have worked with values like 1, 2, 3, so that column will be either correctly set to int or NULL.

AFAIK the tempblock functionality is not used in practice, so I did not want to spent any more time on it.